src/main/drivers/light_ws2811strip.c

```diff
@@ -44,8 +44,11 @@
 
 #if defined(STM32F1) || defined(STM32F3)
 uint8_t ledStripDMABuffer[WS2811_DMA_BUFFER_SIZE];
-#else
+#elif defined(STM32F4) || defined(UNIT_TEST)
```

Comment:
Tests have an empty defined for `FAST_RAM_ZERO_INIT`, no use to mention them there explicitly.
Let's make it `#elif defined(STM32F7)` and `#else`